x-pack/filebeat/module/cisco/ftd/_meta/fields.yml

```diff
@@ -90,3 +90,15 @@
     type: object
     description:
       Raw fields for Security Events.
+  
+  - name: connection_type
```

Comment:
Would it make sense to have common `cisco.connection_type` and `cisco.dap_records` instead of one different field on each fieldset?

Reply:
It makes sense, I have a task in my backlog to review how to do this taking also into account the other cisco fileset (ios), so let's leave it as is for now